pyspeckit/spectrum/models/dcn.py

```diff
@@ -12,7 +12,10 @@
 import astropy.units as u
 
 freq_dict_cen = {
-                'J1-0': 72413.5040e6,
+'''
+Main component from HFS
+'''
+                'J1-0': 72414.9330e6,
```

Comment:
from the CDMS file, I get the following frequencies:
72414.6936 MHz, 144828.0015 MHz, 217238.5378 MHz. 

Please change to these and include the links to the CDMS files with the comments of this file (on top), for this you also need to update the date of retrieval to Today.